jest.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     '\\.svg$': 'jest-raw-loader',
   },
   transformIgnorePatterns: [
-    '/node_modules/(?!(@hashicorp/mktg-logos|@hashicorp/react-|@hashicorp/sentinel-embedded|@hashicorp/nextjs-scripts))',
+    '/node_modules/(?!(@hashicorp/mktg-logos|@hashicorp/react-|@hashicorp/sentinel-embedded|@hashicorp/nextjs-scripts||@hashicorp/platform-))',
```

Comment:
This is legit right?

```suggestion
    '/node_modules/(?!(@hashicorp/mktg-logos|@hashicorp/react-|@hashicorp/sentinel-embedded|@hashicorp/platform-))',
```

Reply:
Still need scripts here as its deep in the dependency tree. I think after a publish here things should get buttoned up.

Reply:
I think the pattern is still wrong here, there are two pipes before the last item. Maybe this doesn't matter though with the matcher?